debian/rules

```diff
@@ -18,7 +18,8 @@ else
 endif
 
 extraopts += -DWITH_OCF=ON -DWITH_LTTNG=ON
-extraopts += -DWITH_PYTHON3=ON -DWITH_MGR_DASHBOARD_FRONTEND=OFF
+extraopts += -DWITH_MGR_DASHBOARD_FRONTEND=OFF
+extraopts += -DWITH_PYTHON3=ON -DWITH_PYTHON2=OFF -DMGR_PYTHON_VERSION=3 -DWITH_MGR_PYTHON3=ON
```

Comment:
i'd recommend:
```sh
extraopts += -DWITH_MGR_DASHBOARD_FRONTEND=OFF
extraopts += -DWITH_PYTHON3=3 -DWITH_PYTHON2=OFF -DMGR_PYTHON_VERSION=3
```

three reasons: 

- group python related options in the same line
- be consistent with specifying python3 versions.
- `WITH_MGR_PYTHON3` is not a valid option at this moment.

Reply:
@alfredodeza i am not sure if you read my comment. i am posting it here again.

Reply:
I've addressed this in the last push

Reply:
Oh, unless this last change is also not what you are looking for?

Reply:
no, i don't think it's addressed. i pulled this change, and checked it locally. i hope i could triple check it.